src/calculateRentalCost.js

```diff
@@ -1,10 +1,18 @@
-/**
+/*
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PriseDay = 40;
+
+  if (days < 3) {
+    return days * PriseDay;
+  } else if (days >= 3 && days < 7) {
+    return days * PriseDay - 20;
+  } else {
+    return days * PriseDay - 50;
+  }
```

Comment:
The function name `calculateRentalCost` is descriptive, which is good. However, the variable `PriseDay` on line 7 is misspelled and should be `PricePerDay` to correctly represent its purpose.